packages/react/modules/components/reconfigure-adapter/reconfigure-adapter.js

```diff
@@ -24,6 +25,11 @@ export class ReconfigureAdapter extends PureComponent<Props> {
     children: null,
   };
 
+  static propTypes = {
+    shouldOverwrite: PropTypes.bool,
+    children: PropTypes.node,
```

Comment:
Feels like having these can't hurt. Another option would be to generate them from flow types but that also feels a bit over done for now.